README.md

```diff
@@ -9,14 +9,40 @@ https://github.com/chrisjshull/homebridge-nest#using-a-google-account
 
 These four items needs to be included into config.json. An example of config.json can be find as _config.json.
 
-## Install
+## Local Install
 After clone the repo, change directory in to the folder.
 ```
 openssl req -x509 -nodes -days 365 -newkey rsa:2048 -keyout ./selfsigned.key -out selfsigned.crt
 npm install
 npm run start
 ```
-open a browser and go to the following url: https://[IP]:5000.
+open a browser and go to the following url: https://127.0.0.1:5000.
+
+Default credentials:
+```
+user: admin
+pass: admin1
+```
+
+## Docker Install
+Build image
+```
+docker build -t nest-camera-site:latest .
+```
+
+Star container:
+```
+docker run --name nest-camera-site -d -p 5000:5000 -v [/path/to/your/nest_folder]:/nest nest-camera-site:latest
```

Comment:
About the port thing. Originally, the port is 5000 by default and you can specify it by changing the environment variable PORT. For example, `PORT=5001 npm run start` will start a server on 5001 port.

Reply:
So, in this case, the port doesn't really matter `-p 5000:5000` is mapping the container's port 5000 to host port 5000, so you can say `-p 5001:5000` in this case you specified your host 5001 port mapped to container's 5000.

Reply:
I've updated the port in Dockerfile as well to 5500